components/ai_chat/core/browser/engine/engine_consumer_claude.cc

```diff
@@ -250,13 +250,17 @@ void EngineConsumerClaudeRemote::GenerateAssistantResponse(
     GenerationDataCallback data_received_callback,
     GenerationCompletedCallback completed_callback) {
   if (conversation_history.empty()) {
-    std::move(completed_callback).Run(base::unexpected(mojom::APIError::None));
+    std::move(completed_callback)
```

Comment:
ditto here (and in all the other engine_consumers) - why is this unexpected no error?